x-pack/plugin/inference/qa/rest/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
+
+dependencies {
+  clusterPlugins project(':x-pack:plugin:inference:qa:test-service-plugin')
+}
+
+restResources {
+  restApi {
+    include '_common', 'bulk', 'indices', 'inference', 'index', 'get', 'update', 'reindex'
+  }
+}
+
+tasks.named('yamlRestTest') {
+  usesDefaultDistribution()
```

Comment:
Is it strictly required that we use the default distribution here? Can these tests simply explicitly install the plugins/modules necessary?

Reply:
I'm not familiar with this. Could you please point me to some docs explaining the process, or examples that don't use the default distribution to check? 🙏